app/views/layouts/application.mobile.haml

```diff
@@ -3,56 +3,52 @@
 -# the COPYRIGHT file.
 
 !!!
-%html{:lang => I18n.locale.to_s, :dir => (rtl?) ? 'rtl' : 'ltr'}
-  %head
+%html{:lang => I18n.locale.to_s, :dir => (rtl?) ? "rtl" : "ltr"}
+  %head{prefix: og_prefix}
     %title
-      = pod_name
+      = page_title yield(:page_title)
 
-    %meta{:name => "description", :content => "diaspora* mobile"}/
-    %meta{:name => "author", :content => "Diaspora, Inc."}/
-    %meta{:charset => 'utf-8'}/
+    %meta{charset:  "utf-8"}/
+    %meta{:name =>"viewport", :content => "width=device-width, minimum-scale=1 maximum-scale=1"}/
```

Comment:
- Use the new Ruby 1.9 hash syntax.
- Surrounding space missing for operator `=>`.